packages/react-native-renderer/src/ReactFabric.js

```diff
@@ -75,8 +71,6 @@ const ReactFabric: ReactNativeType = {
     return ReactPortal.createPortal(children, containerTag, null, key);
   },
 
-  unstable_batchedUpdates: ReactGenericBatching.batchedUpdates,
```

Comment:
If we at some point switch this to be the only ReactNative export, then we might need to add a noop with a warning here just for legacy purposes.